python-gitlab.cfg

```diff
@@ -0,0 +1,9 @@
+[global]
+default = zgtools
+ssl_verify = true
+timeout = 5
+
+[zgtools]
+url = https://gitlab.zgtools.net
+private_token = _K9Y1t8gD9zSKzjYug-x
```

Comment:
What does this token grant access to?

Reply:
It's an API wide read/write token. Looks like the only option Gitlab has. I've regenerated mine now and rebased the commit to remove this file, I added the wrong path to `.gitignore` initially.